great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py

```diff
@@ -24,6 +24,14 @@ class ExpectMulticolumnSumToEqual(MulticolumnMapExpectation):
 
     map_metric = "multicolumn_sum.equal"
     success_keys = ("sum_total",)
+    default_kwarg_values = {
+        "row_condition": None,
+        "condition_parser": None,  # we expect this to be explicitly set whenever a row_condition is passed
+        "ignore_row_if": "all_values_are_missing",
+        "result_format": "BASIC",
+        "include_config": True,
+        "catch_exceptions": True,
```

Comment:
False?

Reply:
@jcampbell Thank you for catching this.  It has been fixed (PR branch updated).  Thanks.